schedmd/slurm/25.11/rockylinux9/Dockerfile

```
@@ -153,15 +153,22 @@ RUN --mount=type=cache,target=/var/cache/dnf,sharing=locked <<EOR
 set -xeuo pipefail
 dnf -q -y install --setopt='install_weak_deps=False' \
   socat \
+  authselect sssd sssd-ad sssd-ldap \
   ./slurm-slurmctld-[0-9]*.rpm \
   ./slurm-[0-9]*.rpm
 rm *.rpm
+# Configure
+sed -i -E "s/^passwd:[[:space:]]+/&slurm /g" /etc/nsswitch.conf
+sed -i -E "s/^group:[[:space:]]+/&slurm /g" /etc/nsswitch.conf
+mkdir -p /etc/authselect
+authselect select sssd with-mkhomedir --force
 EOR
 
 COPY files/etc/supervisord.conf /etc/
 COPY \
   files/etc/supervisord.d/slurmctld.ini \
   files/etc/supervisord.d/fakesystemd.ini \
```

As fakesystemd runs as root by default (), Slurm's `xsystemd_change_mainpid()` call will fail to connect to this socket as-is. This doesn't actually interfere with the ability to reconfigure, but does cause an error message:
```
[2025-12-16T20:47:24] Running as primary controller
[2025-12-16T20:47:24] child started successfully
[2025-12-16T20:47:24] Relinquishing control to new slurmctld process
[2025-12-16T20:47:24] error: xsystemd_change_mainpid: connect() failed for /tmp/fakesystemd.sock: Permission denied
[2025-12-16T20:47:24] Relinquishing control to new child
```
This can be mitigated by using `fakesystemd-slurm.ini`, which was created for the sackd fixes I made for you last week. All this does is runs fakesystemd as `slurm` instead of as `root`.
```suggestion
  files/etc/supervisord.d/fakesystemd-slurm.ini \
```
This results in no error messages on reconfigure:
```bash
[2025-12-16T20:52:47] Running as primary controller
[2025-12-16T20:52:47] child started successfully
[2025-12-16T20:52:47] Relinquishing control to new slurmctld process
[2025-12-16T20:52:47] Relinquishing control to new child
```